package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sol-style-guide",
-  "version": "2.5.0",
+  "version": "2.6.0",
```

Comment:
This bump should have been pushed to develop before branching out to this feature branch.